server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStats.java

```diff
@@ -46,7 +46,6 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             .field(Fields.LOCAL_REFRESH_TIMESTAMP, remoteSegmentUploadShardStats.localRefreshTimeMs)
             .field(Fields.REMOTE_REFRESH_TIMESTAMP, remoteSegmentUploadShardStats.remoteRefreshTimeMs)
             .field(Fields.REFRESH_LAG, remoteSegmentUploadShardStats.localRefreshNumber - remoteSegmentUploadShardStats.remoteRefreshNumber)
```

Comment:
should we get rid of localRefreshNumber and remoteRefreshNumber fields from the stat class and use only the refreshLag?

Reply:
Just kept it to be flexible for other usecases that may comeup in future. If you feel strongly on removing i can update